controller/customdomain_controller.go

```diff
@@ -277,7 +277,9 @@ func (r *CustomDomainReconciler) Reconcile(ctx context.Context, request ctrl.Req
 				return reconcile.Result{}, err
 			}
 			isAWS := *cloudPlatform == "AWS"
-			if isAWS {
+			lbType := customIngress.Spec.EndpointPublishingStrategy.LoadBalancer.ProviderParameters.AWS.Type
+
+			if isAWS && lbType != "NLB" {
```

Comment:
The use of the `"NLB"` string here could perhaps take advantage of referencing the defined constant `AWSNetworkLoadBalancer` instead.

Reply:
Good point. There are also "Classic" strings that can also be replaced for consitency.

Reply:
Good observation! I changed it.